services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/models.py

```diff
@@ -1,29 +1,59 @@
-from typing import Dict, List
+from typing import Optional
 
 from packaging import version
-from pydantic import BaseModel, ByteSize, Field
+from pydantic import BaseModel, ByteSize, Field, validator
 
 LEGACY_INTEGRATION_VERSION = version.Version("0")
 
 
 class ContainerHostConfig(BaseModel):
-    binds: List[str] = Field(
+    # NOTE: https://github.com/ITISFoundation/osparc-simcore/issues/3506
+    # Be careful! --priviledged, --pid=host --cap-add XXX should never be usable here!!
+    # at the moment they are not part of the possible configuration but if they were
+    # to, ensure they are properly validated
+
+    binds: list[str] = Field(
         ..., alias="Binds", description="A list of volume bindings for this container"
     )
     init: bool = Field(
-        True,
+        default=True,
         alias="Init",
         description="Run an init inside the container that forwards signals and reaps processes. This field is omitted if empty, and the default (as configured on the daemon) is used",
     )
     memory: ByteSize = Field(..., alias="Memory", description="Memory limit in bytes")
+    memory_swap: Optional[ByteSize] = Field(
+        default=None,
+        alias="MemorySwap",
+        description="Total memory limit (memory + swap). Set as -1 to enable unlimited swap.",
```

Comment:
In what case would we want unlimited swap? I'm trying to figure out if for the dy-sidecar services this is required.

Reply:
Unlimited swap in a comp/dy service is to the best of my knowledge not a realistic scenario for osparc. For debugging and exploring the behavior of certain services it might be of interest to the developers.

Reply:
I took over what the docker REST API allows. I don't want to re-invent the wheel here.

Reply:
also please remember that the images we run are supposed to be checked beforehand.

Reply:
ok I will disable it.